src/installer/test/Microsoft.NET.HostModel.Tests/Microsoft.NET.HostModel.Bundle.Tests/BundlerConsistencyTests.cs

```diff
@@ -83,10 +83,34 @@ public void TestWithDuplicateEntriesFails()
             Assert.Throws<ArgumentException>(() => bundler.GenerateBundle(fileSpecs));
         }
 
-        [InlineData(true)]
-        [InlineData(false)]
+        [InlineData(BundleOptions.None)]
+        [InlineData(BundleOptions.BundleNativeBinaries)]
+        [InlineData(BundleOptions.BundleAllContent)]
```

Comment:
Add `BundleOtherFiles` as well.

Reply:
Sure, I'll add it. There's no way to trigger that selection from the SDK, so had skipped it.